app/assets/scripts/components/map/legend.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import { Dropdown } from 'openaq-design-system';
+import c from 'classnames';
+
+import { generateLegendStops } from '../../utils/colors';
+
+export default function Legend({ parameters, activeParameter }) {
+  let history = useHistory();
+
+  function onFilterSelect(parameter, e) {
```

Comment:
![image](https://user-images.githubusercontent.com/19439427/102635248-10696180-4121-11eb-89c1-0604891eb185.png)
seeing this error from world map

Reply:
wonder how history can be undefined, thats coming from react router 🤔 I tried, but never saw this error myself

Reply:
must be a rebuild issue or something then